randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1971,17 +1971,16 @@ Extra - room_id: [42]
   > Event - Serris
       All of the following:
           Any of the following:
-              # Serris Weapon Requirements
+              # Serris Weapon Requirements (50HP)
               Charge Beam or Missiles ≥ 10
               Missiles ≥ 5 and 20 DMG Missiles
```

Comment:
not missing missiles should definitely not be trickless

Reply:
bumped the missiles a bit, I don't think it's worth getting more granular than this?